packages/rum/src/domain/record/serialize.ts

```diff
@@ -249,6 +250,10 @@ export function serializeAttribute(
     }
   }
 
+  if (nodePrivacyLevel === NodePrivacyLevel.MASK_USER_INPUT && attributeName === 'placeholder') {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏we might not want that, since it's not a user input?